docs/security-privacy-questionnaire.md

```diff
@@ -0,0 +1,105 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+Payment Link URI is exposed. This includes the scheme (e.g., upi, bitcoin, paypal) and potentially other parameters embedded in the URI, such as payee address, amount, and currency. This information is necessary to identify the payment method and initiate the transaction.
```

Comment:
Who owns this information and who receives it? I assume that the website owns it and "payment clients (browser extensions or integrated wallets)" receive it.

What does the word "potentially" mean in this sentence? I though that the whole URL is exposed, regardless of what it contains.

When you write "initiate" the transaction, does the user get a say in whether to initiate this transaction? Perhaps write "prompt the user to initiate a transaction".

Reply:
> Who owns this information and who receives it? I assume that the website owns it and "payment clients (browser extensions or integrated wallets)" receive it.

Thanks! Mentioned that it's exposed by website. 

> What does the word "potentially" mean in this sentence? I though that the whole URL is exposed, regardless of what it contains.

Removed the word "potentially" as it causes confusion. I was suppose to refer to potential parameters we will have in the URI. 

> When you write "initiate" the transaction, does the user get a say in whether to initiate this transaction? Perhaps write "prompt the user to initiate a transaction".

Done.